scripts/preprocessing/04b-run_ssp.py

```diff
@@ -57,10 +57,14 @@ def run_ssp(*, cfg, subject, session=None):
     logger.debug(**gen_log_kwargs(message=msg, subject=subject,
                                   session=session))
 
-    reject_ecg_ = config.get_ssp_reject(
-        ssp_type='ecg',
-        epochs=(create_ecg_epochs(raw)
-                if cfg.ssp_reject_ecg == 'autoreject_global' else None))
+    reject_ecg_ = None
+    ecg_epochs = None
+    if cfg.ssp_reject_ecg == 'autoreject_global':
+        ecg_epochs = create_ecg_epochs(raw)
+        if len(ecg_epochs) >= 5:
+            reject_ecg_ = config.get_ssp_reject(
+                ssp_type='ecg',
+                epochs=ecg_epochs)
```

Comment:
was it done like this in old @dengemann 's code?
First I would expose the param 5 in the config.py as a study parameter.
Second I would have thought that in this case you rather do not compute
any projection. ecg_projs is therefore here just an empty list.

Reply:
In @dengemann's code it is : 
```
    ecg_epochs = mne.preprocessing.create_ecg_epochs(raw)
    if len(ecg_epochs) >= 5:
        reject_ecg = get_rejection_threshold(ecg_epochs, decim=8)
    else:
        reject_ecg = None
```

Reply:
this means compute an SSP (which can be very bad if not enough data). What I suggest to not compute any SSP.

Reply:
yes that is what has to be done (and the old logic did it).